Feature/declare queues

jmdcal opened the pull request:
Relates to https://github.com/saltstack/salt/issues/26222 uses that and is needed by it to install the management tool

gravyboat commented:
I don't think we can merge this in until your feature is into salt core unless you make it optional. Could you remove these includes from the init, then add comments in the readme (or states directly) noting that this is not available yet?

jmdcal commented:
Ok I have moved the rabbitmqadmin to its own module and added a check for it. Also installing into /usr/sbin so it is next to the rabbitmqctl and in the path. This should address your concerns.

h4ck3rm1k3 commented:
please close this, jmdcal is my work account and now inactive. This will be continued https://github.com/saltstack-formulas/rabbitmq-formula/pull/29 here.
